test/index.js

```diff
@@ -564,6 +565,11 @@ describe('maybe', function() {
                     errorEq(Error, 'Cannot instantiate Maybe'));
     });
 
+    it('has its type definition exported', function() {
+      assert(S.MaybeType($.Number).test(S.Nothing()));
+      assert(S.MaybeType($.Number).test(S.Just(1)));
```

Comment:
I prefer to write `eq(expr, true)` rather than `assert(expr)` as it's more rigorous. Also, it'd be nice to test a value which is not a member of `Maybe Number`.